pkg/server/admin.go

```diff
@@ -761,22 +761,13 @@ func (s *adminServer) getDatabaseStats(
 }
 
 func (s *adminServer) getNumDatabaseIndexRecommendations(
```

Comment:
nit: There's only one caller of `getNumDatabaseIndexRecommendations` which is just essentially len(getDatabaseIndexRecommendations) right now. I'm wondering if it'll make things slightly clearer to get rid of this wrapper and just use `getDatabaseIndexRecommendations` directly at the db details function?

Reply:
Yea makes sense, I've moved it to use `getDatabaseIndexRecommendations` directly